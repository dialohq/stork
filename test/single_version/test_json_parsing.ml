let () =
  Test_utils.print_parsed_json
    ~to_string:Single_version.string_of_employee
    ~from_string:Single_version.employee_of_string
    ~read_from_file:Single_version.read_employee_from_file
    "single_version_1.json"
