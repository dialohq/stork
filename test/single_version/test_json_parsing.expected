{"name":"Paul","age":30,"position":["dev",["Company",{"name":"Dialo","address":"Canary Islands"}]],"version":1,"skills":[["design","typography"],["computer science","ocaml"]]}
---
{"name":"Paul","age":30,"position":["dev",["Company",{"name":"Dialo","address":"Canary Islands"}]],"version":1,"skills":[["design","typography"],["computer science","ocaml"]]}
---
